### PR TITLE
RFECV quick method update

### PR DESCRIPTION
**Pull request by percygautam**
This PR addresses issue #960 
I have made the following changes:

1. Added show hyperparameter to rfecv.
2. Updated test for the quick method
3. Updated documentation

### Sample Code and Plot

```python

from sklearn.ensemble import RandomForestClassifier
from sklearn.model_selection import StratifiedKFold

from yellowbrick.model_selection import rfecv
from yellowbrick.datasets import load_credit

# Load classification dataset
X, y = load_credit()

cv = StratifiedKFold(5)
visualizer = rfecv(RandomForestClassifier(), X=X, y=y, cv=cv, scoring='f1_weighted')
```

### CHECKLIST

- [x] _Is the commit message formatted correctly?_
- [x] _Have you noted the new functionality/bugfix in the release notes of the next release?_

- [x] _Included a sample plot to visually illustrate your changes?_
- [x] _Do all of your functions and methods have docstrings?_
- [x] _Have you added/updated unit tests where appropriate?_
- [ ] _Have you updated the baseline images if necessary?_
- [x] _Have you run the unit tests using `pytest`?_
- [x] _Is your code style correct (are you using PEP8, pyflakes)?_
- [x] _Have you documented your new feature/functionality in the docs?_

- [ ] _Have you built the docs using `make html`?_

### CODE REVIEW

**Comment by pdamodaran** [comment]
Hi @percygautam - thanks a lot for working on adding the quick method for the RFECV visualizer.  This is very close to being merged.  The documentation for this visualizer needs to be updated to include a visualization for the quick method example.  Take a look at my in-line comment for how to do this.  Let me know if you have any questions.

Thanks a lot for submitting this PR!

**Comment by percygautam** [comment]
Hi @pdamodaran I didn't understand the changes which you are suggesting. Moreover, I can't seem to find the inline comments. Can you please suggest me the changes!

**Comment by pdamodaran** [comment]
Hi @percygautam - sorry for the confusion.  If you click on `Files Changed` located below the PR Header on this page, you will see my comment regarding the changes you made to `docs/api/model_selection/rfecv.rst`.  

What I had mentioned there was that the example you provided for the quick method does not contain the image that is generated when the code you provided is run.

In order for the image to be generated, you will need to do the following:

Place the image in the `docs/api/model_selection/images` folder.  Give the image a meaningful name such as `rfecv_quick_method.png`.

Include a link to this image in the rst file such as this:

`.. image:: images/rfecv_quick_method.png`

In addition to the above, copy the code sample in the `docs/api/model_selection/rfecv.py` file.

Take a look at the other examples in this file to guide you.

After you are done with these changes, test your changes by following the instructions provided in this link:

https://github.com/DistrictDataLabs/yellowbrick/blob/develop/docs/contributing/developing_visualizers.rst#building-the-docs

Let me know if you have any more questions.

**Comment by percygautam** [comment]
Hi @pdamodaran , I have done the changes you asked, but it seems there are some conflicts due to my pull request. Kindly, suggest the solution.

File: docs/api/model_selection/rfecv.rst [hunk 1]
```diff
@@ -63,10 +70,29 @@ In this example we can see that 19 features were selected, though there doesn't
...
+    from yellowbrick.model_selection import rfecv
+    from yellowbrick.datasets import load_credit
+
+    # Load classification dataset
+    X, y = load_credit()
+
+    cv = StratifiedKFold(5)
+    visualizer = rfecv(RandomForestClassifier(), X=X, y=y, cv=cv, scoring='f1_weighted')
+
+
```
[the remark this answers is not in the document]
Comment (percygautam) [reply docs/api/model_selection/rfecv.rst]: Thanks @pdamodaran , I have added the image in the said folder and also added the link to the image.

**Comment by percygautam** [comment]
Hi @pdamodaran , thanks for the detailed review. I have done the changes requested, but I have some doubts.
First, the added quick method function in the rfecv.py file in the API directory will have some discrepancy as we need to pass the parameter _'outpath'_ to show method which is not possible in quick method. 
Also, my IDE is not rendering images in the restructured text(rst) and just showing the path name. Also it is showing unknown directive type _'automodule'_ and _'seealso'_. I think it has something to do with `conf.py` file. Can you please help me with this!

**Comment by naresh-bachwani** [comment]
Hey there @percygautam,
Great work on the PR so far! I think that I can answer some of your questions.
1. The quick method in `docs/model_selection/rfecv.py` looks fine to me. @pdamodaran can tell if there is any other possibility.
2. For your second question I suggest you follow this[link](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/docs/contributing/developing_visualizers.rst#building-the-docs).
I currently don't know about any IDE which renders images from code in rst. The above method works for me.

**Comment by pdamodaran** [comment]
@percygautam - thanks for making the additional changes to the API doc to include the quick method and thanks @naresh-bachwani for answering @percygautam's questions.  I made one small change and everything looks good.  I will approve this PR and merge this in.  @percygautam if you still have questions regarding how to render the images in the API docs please feel free to reach out!

**Comment by percygautam** [comment]
Thanks, @pdamodaran, and @naresh-bachwani for answering my questions. The method suggested for rendering images to API doc works fine. I think the problem is with IDE not supporting the full features of sphinx.

**Pull request merged**